yaas/parsers/service.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import inspect
+
+from .. import service
+from .. import config
+from .. import utils
+
+def define_subcommand(parent_parser):
```

Comment:
Why? In a normal argparse setup, defining all parsers/arguments would be in one giant file. We've just broken up the logic of adding subparsers and arguments in a more modular way.